### PR TITLE
Added sequence reference checks to OpenRA.Mods.Cnc.Traits.Render

### PR DESCRIPTION
**Pull request by Mailaender**
A followup of https://github.com/OpenRA/OpenRA/pull/7709.

### CODE REVIEW

**Comment by abcdefg30** [comment]
👍

**Comment by GraionDilach** [comment]
:+1:

File: OpenRA.Mods.Cnc/Traits/SupportPowers/IonCannonPower.cs [hunk 1]
```diff
@@ -27,8 +27,12 @@ class IonCannonPowerInfo : SupportPowerInfo, IRulesetLoaded
 		[Desc("Amount of time to keep the camera alive")]
 		public readonly int CameraRemoveDelay = 25;
 
-		[Desc("Effect sequence to display")]
+		[Desc("Effect sequence sprite collection")]
```
Comment (pchote) [inline OpenRA.Mods.Cnc/Traits/SupportPowers/IonCannonPower.cs]: Please change the description to "Effect image" to match everything else.  There's no need to introduce a new term for them.

Comment (Mailaender) [inline OpenRA.Mods.Cnc/Traits/SupportPowers/IonCannonPower.cs]: Done.

**Comment by abcdefg30** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-%28bleed%29/_compare/27693574e6179b447a769f725292f97831fa869d...b12c933a5c5211ebb105086d6c5b730c34f94ede)

**Pull request merged**